style.css

```diff
@@ -0,0 +1,727 @@
+  background-color: #ff6b00;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: none;
+  margin-left: 142px;
+}
+
+.about-button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+}
+
+.section-about hr {
+  margin-left: 141px;
+}
+
+.about-language > ul li::before {
+  content: "\2022";
+  color: #ff6b00;
+  line-height: 100px;
+  font-size: 32px;
+  margin-left: 12px;
+  position: relative;
+  left: -10px;
+}
+
+.about-language > hr {
+  width: 400px;
+}
+
+.about-language ul {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+}
+
+.about-language ul li {
+  content: "\2022";
+  color: white;
+  line-height: 100px;
+  font-size: 32px;
+  margin-left: 12px;
+  left: -10px;
+}
+
+.sublist {
+  display: flex;
+  flex-direction: column;
+}
+
+@media only screen and (max-width: 600px) {
+  .about-button {
+    margin-left: 24px;
+  }
+
+  .about-text p {
+    margin-left: 24px;
+  }
+
+  .about-text h2 {
+    margin-left: 24px;
+  }
+
+  .section-about hr {
+    margin-left: 24px;
+  }
+
+  .about-language ul {
+    flex-direction: column;
+  }
+}
+
+@media only screen and (max-width: 900px) {
+  .about-button {
+    margin-left: 24px;
+  }
+
+  .about-text p {
+    margin-left: 24px;
+  }
+
+  .about-text h2 {
+    margin-left: 24px;
+  }
+
+  .section-about hr {
+    margin-left: 24px;
+  }
+
+  .about-language ul {
+    flex-direction: column;
+  }
+}
+
+.section-contact {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  justify-content: space-around;
+  align-items: center;
+  background: #e5e5e5;
+  height: auto;
+  margin: 0 24px;
+}
+
+.contact-text p {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #262626;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-me h2 {
+  display: block;
+  margin-left: 24px;
+  font-weight: bold;
+  color: #172b4d;
+  text-align: center;
+  margin-top: 120px;
+  margin-bottom: 20px;
+}
+
+input {
+  border-width: 0 0 2px;
+  border-color: #dbd8d7;
+  background-color: #e5e5e5;
+  height: 18px;
+  width: 295px;
+  margin-top: 39px;
+}
+
+input::placeholder {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+textarea {
+  border: none;
+  width: 327px;
+  height: 114px;
+  margin-top: 36px;
+  margin-left: 24px;
+  background: #fbf8f7;
+}
+
+textarea::placeholder {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
+
+.button {
+  background-color: #ff6b00;
+  color: #fff;
+  width: 131px;
+  height: 48px;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.button:hover {
+  background: #fff5e1;
+  border: 1px solid #f55800;
+  color: #f55800;
+  box-sizing: border-box;
+}
+
+@media only screen and(max-width: 600px) {
```

Comment:
Please use the value for min-width and max-width property in range of 992px - 1100px.